perception/simple_object_merger/test/test_simple_object_merger_node.cpp

```diff
@@ -0,0 +1,149 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "simple_object_merger/simple_object_merger_node.hpp"
+
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <autoware_test_utils/autoware_test_utils.hpp>
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+using autoware_perception_msgs::msg::DetectedObject;
+using autoware_perception_msgs::msg::DetectedObjects;
+using autoware_perception_msgs::msg::ObjectClassification;
+using simple_object_merger::SimpleObjectMergerNode;
+
+std::shared_ptr<autoware::test_utils::AutowareTestManager> generateTestManager()
+{
+  auto test_manager = std::make_shared<autoware::test_utils::AutowareTestManager>();
+  return test_manager;
+}
+
+std::shared_ptr<SimpleObjectMergerNode> generateNode()
+{
+  auto node_options = rclcpp::NodeOptions{};
+  const auto simple_object_merger_dir =
+    ament_index_cpp::get_package_share_directory("simple_object_merger");
+  node_options.arguments(
```

Comment:
the test result can be changed by this configuration file.
May its better to have a separated configuration file or set the parameters here.

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/7637/commits/cd92ee108ba586199b7ade0e39281311e22cb673